full-service/src/json_rpc/v2/models/masked_amount.rs

```diff
@@ -54,14 +54,17 @@ impl TryFrom<&MaskedAmount> for mc_transaction_core::MaskedAmount {
             &hex::decode(&src.commitment)
                 .map_err(|err| format!("Could not decode hex for amount commitment: {:?}", err))?,
         );
-        Ok(Self {
+
+        let masked_amount_v1 = mc_transaction_core::MaskedAmountV1 {
```

Comment:
The v2 API should plan to support MaskedAmountV2...

Reply:
agreed, will add that in 👍